pychatml/alpaca_converter.py

```diff
@@ -0,0 +1,55 @@
+from typing import Dict, List
+
+from pychatml.chatml_converter import ChatMLConverter
+
+
+class AlpacaChatMLConverter(ChatMLConverter):
```

Comment:
maybe lets call it just `Alpaca` since it comes out of the `pychatml.chatml_converter` package it should be clear what it does